src/tls.c

```diff
@@ -2984,6 +2984,7 @@ static int TLSX_CSR_Parse(WOLFSSL* ssl, byte* input, word16 length,
                                                                  byte isRequest)
 {
     int ret;
+    DecodedCert* cert;
```

Comment:
If configured with "--disable-tls13", this variable `cert` is unused. I'd declare it down in the block where it is used, right before the XMALLOC.